projects/frontend/shared-components/gui/.editorconfig

```diff
@@ -3,11 +3,14 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = space
+indent_style = tab
```

Comment:
Why change this? I think spaces is better, due to some operating systems/editors handling tabs differently.

Reply:
It was with space in .editorconfig, but prettier that is used as plugin in pre-commit, was using tabs in its own config, and pre-hook auto-format was applying tabs instead of spaces.
I prefer spaces too, but that will have implications on all files, because plugin was not configured properly before.
see file: `projects/frontend/shared-components/gui/.prettierrc.json`

Reply:
I just revert it back with space and I committed all shared-components directory files with the new formatting with spaces.